src/NzbDrone.Core/Indexers/Events/IndexerDownloadEvent.cs

```diff
@@ -1,26 +1,37 @@
 using NzbDrone.Common.Messaging;
+using NzbDrone.Core.Parser.Model;
 
 namespace NzbDrone.Core.Indexers.Events
 {
     public class IndexerDownloadEvent : IEvent
     {
-        public int IndexerId { get; set; }
+        public ReleaseInfo Release { get; set; }
         public bool Successful { get; set; }
         public string Source { get; set; }
         public string Host { get; set; }
         public string Title { get; set; }
         public bool Redirect { get; set; }
         public string Url { get; set; }
+        public int DownloadClientId { get; set; }
+        public string DownloadClient { get; set; }
+        public string DownloadClientName { get; set; }
+        public string DownloadId { get; set; }
+        public GrabTrigger GrabTrigger { get; set; }
 
-        public IndexerDownloadEvent(int indexerId, bool successful, string source, string host, string title, string url, bool redirect = false)
+        public IndexerDownloadEvent(ReleaseInfo release, bool successful, string source, string host, string title, string url)
         {
-            IndexerId = indexerId;
+            Release = release;
             Successful = successful;
             Source = source;
             Host = host;
             Title = title;
-            Redirect = redirect;
```

Comment:
is dropping redirect here intentional?

Reply:
Yes, it gets set manually for the one instance it's used